src/lando/main/models/profile.py

```diff
@@ -59,20 +85,22 @@ def get_all_scm_permissions(cls) -> dict[str, Permission]:
         return permissions
 
     @property
-    def phabricator_api_key(self):
-        # Temporary placeholder for phabricator_api_key field.
-        # See https://bugzilla.mozilla.org/show_bug.cgi?id=1899397.
-        return ""
-
-    def _has_scm_permission_groups(self, codename, groups):
-        """Return whether the group membership provides the correct permission.
-
-        In order to have a particular SCM permission, both the "active" and "all" groups
-        need to exist, and the "expired" group should not exist.
-        """
-        return {f"all_{codename}", f"active_{codename}"}.issubset(
-            groups
-        ) and f"expired_{codename}" not in groups
+    def phabricator_api_key(self) -> str:
+        """Decrypt and return the value of the Phabricator API key."""
+        encrypted_key = bytes(self.encrypted_phabricator_api_key)
```

Comment:
Question: why is this wrapped in a `bytes()` call?

Reply:
Good question. `BinaryField` returns a [memoryview](https://docs.python.org/3.6/library/stdtypes.html#memoryview) which I convert here to `bytes`. I think this is because I'm using psycopg2, but we should probably start using psycopg3, which takes care of this issue. I think this is out of scope for this PR, but perhaps within scope for prod readiness.